### PR TITLE
Compiler passes section collab

### PR DESCRIPTION
**Pull request by indie**
Updating the section on compiler passes to collaborate with @jbobba and others on how to better present this info.

### CODE REVIEW

**Comment by indie** [comment]
Adding @rkimballn1  to review the latest changes on `165500d5d5341c6ac803fca3f3a6536f3074e10e` since these are from the presentation, and I'm unsure if they were affected by the new backend API changes.

File: doc/sphinx/source/buildlb.rst [hunk 1]
```diff
@@ -62,7 +66,7 @@ as needed:
    -- NGRAPH_CODE_COVERAGE_ENABLE:     OFF
    -- NGRAPH_LIB_VERSIONING_ENABLE:    OFF
    -- NGRAPH_PYTHON_BUILD_ENABLE:      OFF
-   -- NGRAPH_USE_PREBUILT_LLVM:        OFF
+   -- NGRAPH_USE_PREBUILT_LLVM:        FALSE
```
Comment (rkimballn1) [inline doc/sphinx/source/buildlb.rst]: I don't understand this change. OFF was good.

Comment (indie) [reply doc/sphinx/source/buildlb.rst]: I have no idea how this happened.  Will revert.

**Comment by indie** [comment]
@jbobba and I worked on this via email, so he requested to be removed from PR

**Pull request merged**